src/components/dapp-staking/my-staking/WasmAdArea.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="wrapper--card">
+    <div
+      class="wrapper--img"
+      :style="`background-image: url('${bg_img.wasm_hero}')`"
+      @click="goToLink()"
```

Comment:
Could you advise where to click to open the docs page? I've clicked on the marked area but nothing has happened.

![image](https://user-images.githubusercontent.com/92044428/227453389-87771526-f51e-4ac0-9541-ebcbd7ede1fd.png)

Reply:
![image](https://user-images.githubusercontent.com/92044428/227453742-93320a2d-f4d9-4e61-a54f-bb0e390f82a4.png)

Reply:
of course, WASM ads area. it seems working well on my side. can you clear cache and trying again?

Reply:
Yes I have checked and looks good thanks! I'll update as soon as I heard things from the team.

Reply:
@sirius651 Ok, this is interesting. It works on Chrome but not on FireFox.

Reply:
> It works on Chrome but not on FireFox.

It worked after clearing cache on FireFox.

Reply:
Hm which version of FF are you using? In my FF, it works well.

Reply:
111.0 (64-bit)

<img width="528" alt="image" src="https://user-images.githubusercontent.com/92044428/227462771-be835eb3-8ba5-44ce-b0db-bfdcdba4ccf0.png">